uppercase_vowels/exercise.md

```diff
@@ -0,0 +1,34 @@
+---
+Level (beginner, intermediate, advanced): beginner
+Tags (2-5): string
+Dependencies (list any required libraries or leave as "None"):
+GitHub user to credit: xettrisomeman
+---
+
+# Uppercase Vowels
+
+---
+ The Python function `uppercase_vowels` takes in `string` or `filename` and returns `string` with vowels uppercased. The `string` can contain punctuation, uppercase consonants but in the final result vowels should be uppercased and consonants should be lowercased and punctuations should be removed while preserving spaces.
```

Comment:
nice

Reply:
we do have two vowel exercises but they are different:

<img width="303" alt="image" src="https://github.com/user-attachments/assets/6ef0ae5c-a8c5-4219-b286-f995262a1d02">